ExcelToCWMS/ProcessDataTable.cs

```diff
@@ -4,13 +4,12 @@
 using System.Data;
 using System.Linq;
 using System.Text.RegularExpressions;
-using TimeUtilities;
 
 namespace ExcelToCWMS
 {
     public class ProcessDataTable
     {
-        public static TimeSeries[] GetTimeSeriesFromExcel(string filename, string sheetName, DateTime startTime, DateTime endTime, TimeSpan offset)
+        public static TimeSeries[] GetTimeSeriesFromExcel(string filename, string sheetName, DateTime startTime, DateTime endTime, TimeZoneInfo tzInfo)
```

Comment:
How about supporting a user that doesn't want to bother with startTime, endTime and tzInfo.  Can we make some smart defaults.   perhaps overloaded.
    public static TimeSeries[] GetTimeSeriesFromExcel(string filename, string sheetName)
    {
      DateTime startTime = DateTime.MinValue;
      DateTime endTime = DateTime.MaxValue;
      TimeZoneInfo tzInfo = TimeZoneInfo.Local;
      return GetTimeSeriesFromExcel(filename, sheetName);
    }

Reply:
note: there are a couple methods below that could be private.